code/style.css

```diff
@@ -97,31 +139,36 @@ p {
 }
 
 /*Desktop version - KOLLA UPP STRL */
+@media (min-width: 1025px) {
```

Comment:
Ok, now I feel completely dumb because I already have margin-top but I didn't play around with it (just before hand-in on Sunday I got the problem with the footer so I just felt too stressed) ie changing it to eg your suggestion, 50 px. I just let it stay with 20 px and couldn't understand why it behaved the way it did in different viewports. 😅 Thanks Sherin, 50px is the perfect size for my footer and I will go with that. 🥳 Sometimes a new set of eyes really can do magic 👀🙌🏽